ui/src/main/webapp/src/app/executions/executions-layout.component.ts

```diff
@@ -117,6 +117,16 @@ export class ExecutionsLayoutComponent extends ProjectLayoutComponent implements
                 this.execution,
                 'migration-issues'
             ),
+            new ContextMenuItem(
+                'Static Reports',
+                'fa-external-link',
+                true,
+                WindupExecutionService.formatStaticReportUrl(this.execution),
+                null,
+                null,
+                '_blank',
+                true
+            )
```

Comment:
This one should check if static reports are generated. Also it uses different icon than other links.
(I used icon `fa-window-restore` which seemed to me little bit more appropriate than `fa-external-link` - mainly because I think that icon is widely used for external links, which has little bit different meaning - it usually leads to some external website. It does is not indicate link will be opened in new window).
  
http://fontawesome.io/icon/window-restore/
http://fontawesome.io/icon/external-link/